demos/python_demos/whiteboard_inpainting_demo/utils/network_wrappers.py

```diff
@@ -0,0 +1,123 @@
+        image_height, image_width = frame.shape[:2]
+        scale = min(self.h / image_height, self.w / image_width)
+        processed_image = cv2.resize(frame, None, fx=scale, fy=scale)
+        processed_image = processed_image.astype('float32').transpose(2, 0, 1)
+
+        sample = dict(original_image=frame,
+                      meta=dict(original_size=frame.shape[:2],
+                                processed_size=processed_image.shape[1:3]),
+                      im_data=processed_image,
+                      im_info=np.array([processed_image.shape[1], processed_image.shape[2], 1.0], dtype='float32'))
+        return sample
+
+    def forward(self, im_data, im_info):
+        if (self.h - im_data.shape[1] < 0) or (self.w - im_data.shape[2] < 0):
+            raise ValueError('Input image should resolution of {}x{} or less, '
+                             'got {}x{}.'.format(self.w, self.h, im_data.shape[2], im_data.shape[1]))
+        im_data = np.pad(im_data, ((0, 0),
+                                   (0, self.h - im_data.shape[1]),
+                                   (0, self.w - im_data.shape[2])),
+                         mode='constant', constant_values=0).reshape(1, self.c, self.h, self.w)
+        im_info = im_info.reshape(1, *im_info.shape)
+        output = self.net.net.infer(dict(im_data=im_data, im_info=im_info))
+
+        classes = output['classes']
+        valid_detections_mask = classes > 0
+        classes = classes[valid_detections_mask]
+        boxes = output['boxes'][valid_detections_mask]
+        scores = output['scores'][valid_detections_mask]
+        masks = output['raw_masks'][valid_detections_mask]
+        return boxes, classes, scores, np.full(len(classes), 0, dtype=np.int32), masks
+
+    def get_detections(self, frames, return_cropped_masks=False, only_class_person=True):
+        outputs = []
+        for frame in frames:
+            data_batch = self.preprocess(frame)
+            im_data = data_batch['im_data']
+            im_info = data_batch['im_info']
+            meta = data_batch['meta']
+
+            boxes, classes, scores, _, masks = self.forward(im_data, im_info)
+            scores, classes, boxes, masks = postprocess(scores, classes, boxes, masks,
+                                                        im_h=meta['original_size'][0],
+                                                        im_w=meta['original_size'][1],
+                                                        im_scale_y=meta['processed_size'][0] / meta['original_size'][0],
+                                                        im_scale_x=meta['processed_size'][1] / meta['original_size'][1],
+                                                        full_image_masks=True, encode_masks=False,
+                                                        confidence_threshold=self.confidence)
+            frame_output = []
+            for i in range(len(scores)):
+                if only_class_person and classes[i] != 1:
```

Comment:
Please use a named constant instead of just `1`.

Reply:
Logic was reimplemented with list of named classes which will be hided using text files with labels according to a model.